libs/langchain/langchain/llms/azureml_endpoint.py

```diff
@@ -71,11 +84,11 @@ def format_response_payload(self, output: str) -> str:
                     response_json = json.loads(output)
                     return response_json[0]["0"]
     """
-    content_type: Optional[str] = "application/json"
```

Comment:
is this change backwards comptabile? doesnt seem so

Reply:
yes, "content_type" and "accepts" have never been effective in this code, so can be removed